File: cloud/dao/pay_wallet_transaction.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// PayWalletTransaction 会员钱包流水表 pay_wallet_transaction
type PayWalletTransaction struct {
	Id         *int64        `db:"id" json:"id"`                  //bigint 编号,PRI
	WalletId   *int64        `db:"wallet_id" json:"walletId"`     //bigint 会员钱包 id
	BizType    *int32        `db:"biz_type" json:"bizType"`       //tinyint 关联类型
	BizId      *string       `db:"biz_id" json:"bizId"`           //varchar 关联业务编号
	No         *string       `db:"no" json:"no"`                  //varchar 流水号
	Title      *string       `db:"title" json:"title"`            //varchar 流水标题
	Price      *int64        `db:"price" json:"price"`            //bigint 交易金额, 单位分
	Balance    *int64        `db:"balance" json:"balance"`        //bigint 余额, 单位分
	Deleted    *int32        `db:"deleted" json:"deleted"`        //tinyint 删除
	TenantId   *int64        `db:"tenant_id" json:"tenantId"`     //bigint 租户
	Creator    null.String   `db:"creator" json:"creator"`        //varchar 创建人
	CreateTime null.DateTime `db:"create_time" json:"createTime"` //datetime 创建时间
	Updater    null.String   `db:"updater" json:"updater"`        //varchar 更新人
	UpdateTime null.DateTime `db:"update_time" json:"updateTime"` //datetime 更新时间
}

File: admin/src/api/interface/payWalletTransaction.ts
// pay_wallet_transaction 会员钱包流水表
import { ReqPage } from "./index";
export namespace PayWalletTransaction {
  export interface ReqPayWalletTransactionList extends ReqPage {
    deleted?: number; // 删除
    bizType?: number; // 关联类型
    bizId?: string; // 关联业务编号
    no?: string; // 流水号
    title?: string; // 流水标题
  }
  export interface ResPayWalletTransactionItem {
    id: number; // 编号
    walletId: number; // 会员钱包 id
    bizType: number; // 关联类型
    bizId: string; // 关联业务编号
    no: string; // 流水号
    title: string; // 流水标题
    price: number; // 交易金额, 单位分
    balance: number; // 余额, 单位分
    deleted: number; // 删除
    tenantId: number; // 租户
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}
